configure.ac

```diff
@@ -66,10 +66,24 @@ AC_ARG_ENABLE(settings,
               [enable_settings="yes"])
 AM_CONDITIONAL([BUILD_SETTINGS],[test "x$enable_settings" = "xyes"])
 
+# allow systemdunitdir to be overridden on the command line so
+# that make distcheck doesn't fail when run as a non-root user
+m4_define([no_systemdunitdir_error],
+          [m4_normalize([Could not get systemdsystemunitdir setting from]
```

Comment:
I have never seen `m4_normalize` before.

Reply:
Copy pasta from eos-boot-helper. :P I skipped it until make distcheck failed to write to /lib then I realised what it was for...